scripts/docs-update-maps.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -e
+set -o pipefail
+set -u
+
+function step { >&2 echo -e "\033[1m\033[36m* $@\033[0m"; }
+function finish { >&2 echo -en "\033[0m"; }
+trap finish EXIT
+
+OUTPUT="/tmp/`uuidgen`"
+
+step "Updating mapbox-gl-native-ios repository…"
+git fetch --all
+git fetch --tags
+
+VERSION=${1}
```

Comment:
Because the release tag is formatted as `ios-vX.X.X`, we should probably remove the `ios-v` at this point.